.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third_party/p4-pdpi"]
```

Comment:
I'm fine keeping this as one PR this time, but there are several clearly independent things in this CL. Please break them up into separate, smaller PRs. E.g. adding the dependency, enum in old protos, adding new protos could all be separate.

Reply:
Yeah I realized this only after I had done it and looked at the PR.